po/ru.po

```diff
@@ -100,6 +100,7 @@ msgid "song selected"
 msgid_plural "songs selected"
 msgstr[0] "песня выбрана"
 msgstr[1] "песни выбраны"
+msgstr[1] "песни были выбраны"
```

Comment:
Shouldn't the added line be `msgstr[2]` instead of `msgstr[1]`?

Reply:
Shit, I need give more attention. Fixed rn